src/lib/Types/Core.hs

```diff
@@ -42,7 +42,7 @@ import Util (FileHash)
 
 import Types.Primitives
 import Types.Source
-import Types.Imp
+import Types.Imp ( IExpr(ILit), IType, ImpFunName, ImpFunction )
```

Comment:
Why the extra imports?

Reply:
Oops, I think these were automatically added.

Reply:
Fixed in #1070